tools/version-bumper/config.go

```diff
@@ -205,10 +206,11 @@ func filterToStableVersions(input []string) []string {
 	return output
 }
 
-func isPreviewVersion(version string) bool {
+func isStableVersion(version string) bool {
```

Comment:
if we're renaming this function (so it's more accurate) we'll want to flip the true/false below, since right now this is returning _that_ it's a Preview API?